_posts/2022-12-12-november-rubygems-updates.md

```diff
@@ -0,0 +1,84 @@
+---
+title: November 2022 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in November.
+
+## RubyGems News
+This month in RubyGems, we released new versions of RubyGems [3.3.25](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3325--2022-11-02), [3.3.26](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3326--2022-11-16) and Bundler [2.3.25](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2325-november-2-2022), [2.3.26](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2326-november-16-2022).
+
+The following improvements and fixes are also included in these releases (see the changelog for more information):
+
+- avoid installing everything under `lib` at the top in all test files - [#6074]( https://github.com/rubygems/rubygems/pull/6074).
+- fixed broken link in `bundle-platform` man page - [#6071](https://github.com/rubygems/rubygems/pull/6071).
+- added permission restrictions to GitHub actions - [#6081](https://github.com/rubygems/rubygems/pull/6081).
+- removed reference to RVM documentation in the message returned when you run `bundler outdated` - [#6083](https://github.com/rubygems/rubygems/pull/6083).
+
+```bash
+$ bundle outdated
+Could not load OpenSSL.
+You must recompile Ruby with OpenSSL support or change the sources in your Gemfile from 'https' to 'http'. Instructions for compiling with OpenSSL using
+RVM are available at rvm.io/packages/openssl.
+```
+- added a specification to ensure that global gemspecs do not confuse Bundler - [#6086](https://github.com/rubygems/rubygems/pull/6086).
+- fixed issue that occurs when a lockfile gem does not resolve on the current platform - [#6070](https://github.com/rubygems/rubygems/pull/6070).
+- updated the docs for `gemfile` man page - [#6007](https://github.com/rubygems/rubygems/pull/6007).
+
+![Gemfile Man](https://i.imgur.com/veOa0z4.png)
+
```

Comment:
I'd say we don't need the picture here either, the one liner with a link to the PR seems fine.

Reply:
Yes, this was suggested by Irene. We wanted to highlight updates with screenshots or images of what changed. Assuming this doesn't fit, I'll move them to a different section linking back to the one-liner.